README.md

```diff
@@ -1,2 +1,17 @@
 # scalastyle_config
-The Scalastyle configuration we use for our in-house Scala projects
+
+The [Scalastyle](http://www.scalastyle.org/sbt.html) configuration we use for our in-house Scala projects.
+
+# Using with the Scalastyle SBT plugin
+
+Assuming your project already has the [plugin](http://www.scalastyle.org/sbt.html) setup, update your `build.sbt` to include the following settings:
```

Comment:
Maybe add a note that if you have `zendesk_sbt_base_settings` plugin in your project, it is not necessary to do this? (After that is implemented)

Reply:
Will do.